ignite/services/network/networkchain/prepare.go

```diff
@@ -107,12 +132,25 @@ func (c Chain) buildGenesis(ctx context.Context, gi networktypes.GenesisInformat
 		return errors.Wrap(err, "genesis of the blockchain can't be read")
 	}
 
-	// set chain id
-	if err := cosmosutil.SetChainID(genesisPath, c.id); err != nil {
-		return errors.Wrap(err, "chain id cannot be set")
+	if chainID == "" {
+		chainID = cosmosutil.SPNChainID
```

Comment:
cosmosutil cannot be aware of SPN.

network package does not need to assume SPN's chain id. Can we set `spn-1` as the default value for the `--spn-chain-id` flag and refactor places to apply this change?

Reply:
this condition was removed, this is the reason I marked it as solved

Reply:
it's not necessary anymore